### ISSUE TITLE
Amplify Vue Sample getting no userPool error

### ISSUE THREAD

**Issue by timheuer** [aws-amplify/amplify-js#1518]
Cloned the example: https://github.com/aws-samples/aws-amplify-vue
and running the instructions after running the app and trying to create a new user, the console is throwing `no userPool` error.

**Comment by timheuer** [comment]
@richardzcode

**Comment by elorzafe** [comment]
Hi @timheuer  I just tried the instructions and is working for me. Did you successfully run `awsmobile push` after enabing user-signin and user-files?

**Comment by timheuer** [comment]
Yes, just tried again.  I'm following step-by-step on the GitHub instructions.  I'm on Windows.  
![image](https://user-images.githubusercontent.com/4821/44749863-aac4b780-aac8-11e8-9fce-47a11726af18.png)

**Comment by timheuer** [comment]
Also curious why these instructions are using awsmobile -- isn't amplify CLI the thing to use?

**Comment by Zilvinaskli** [comment]
It was working with awsmobile for me. I upgraded it to amplify and I'm getting the same NoUserPool error now and aws-exports.js has only region defined now.

**Comment by elorzafe** [comment]
@timheuer we will update this sample to make it compatible with amplify-cli. I saw "No User Pool" when there are multiple instances of amplify installed (nested node_modules inside of other package)

**Comment by Zilvinaskli** [comment]
Maybe you could provide any tips how to migrate from awsmobile to amplify?

**Comment by mlabieniec** [comment]
@Zilvinaskli we don’t have any way currently to migrate an existing Mobile Hub (what the AWS Mobile ClI uses) project to a cloudformation project. We do have on our roadmap functionality to allow usage of existing AWS resources which should then allow you to use the AWS resources that Mobile Hub provisoned for you. I don’t have a timeline for that yet but we can definitely track this as a +1 to priority on that.

**Comment by questsin** [comment]
problem sill happening after push getting errors
Following resources failed

Resource Name: cognito4bde4608_sns-role (AWS::IAM::Role)
Event Type: create
Reason: Resource creation cancelled


× An error occurred when pushing the resources to the cloud
